meta-resin-common/recipes-devtools/e2fsprogs/e2fsprogs_%.bbappend

```diff
@@ -1,2 +1,12 @@
 ALTERNATIVE_${PN} += "mke2fs"
 ALTERNATIVE_LINK_NAME[mke2fs] = "${base_sbindir}/mke2fs"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/os-files:"
+SRC_URI += "file://e2fsck.conf"
+
+do_install_append() {
+	install -m 644 ${WORKDIR}/e2fsck.conf ${D}${sysconfdir}
+}
+
+CONFFILES_${PN} += "${sysconfdir}/e2fsck.conf"
```

Comment:
We don't use a package manager system so we can ignore this.

Reply:
yea. Just more future proof and just in case.
Harmless as it stands.